File: applications/NCAR_transport/all/mapc2m.f
      subroutine mapc2m(xc,yc,xp,yp,zp)
      implicit none

      double precision xc,yc,xc1,yc1,xp,yp,zp
      logical isdisk, iscart, issphere

      xc1 = 2*xc - 1.d0
      yc1 = 2*yc - 1.d0

      if (iscart()) then
         call mapc2m_cart(xc1,yc1,xp,yp,zp)
      elseif (isdisk()) then
         call mapc2m_disk(xc1,yc1,xp,yp,zp)
      elseif (issphere()) then
         call mapc2m_sphere(xc1,yc1,xp,yp,zp)
      endif

c      call scale_map(xp,yp,zp)
c      call rotate_map(xp,yp,zp)

      end

File: applications/amrclaw/euler/2d/shockbubble/mapc2m.m
function [xp,yp,zp] = mapc2m(xc,yc)

global map isflat;

isflat = true;
s = 0.0;
[xc1,yc1,~] = mapc2m_brick(xc,yc,s);
[xp,yp,zp] = mapc2m_cart(xc1,yc1);
xp = xp + 1;
yp = (yp + 1)/4;

zp = zeros(size(xp));

end